src/openassetio-core/include/openassetio/hostApi/ManagerFactory.hpp

```diff
@@ -219,6 +220,55 @@ class OPENASSETIO_CORE_EXPORT ManagerFactory final {
       const ManagerImplementationFactoryInterfacePtr& managerImplementationFactory,
       const log::LoggerInterfacePtr& logger);
 
+  /**
+   * Creates the default @fqref{hostApi.Manager} "Manager" as defined by
+   * the given TOML configuration file.
+   *
+   * This allows deployments to centralize OpenAssetIO manager settings,
+   * and for hosts to instantiate this manager without the need for
+   * their own settings and persistence mechanism.
+   *
+   * @note This mechanism should be the default approach for a host to
+   * initialize the API. Extended functionality to override this
+   * configuration can optionally be provided, but the ability to use
+   * the shared, default configuration is always required.
```

Comment:
[fix] Be good to remove this from this overload - as the intention of the env var one being the default  mechanism means that no additional configuration is required in supported applications. As this requires an explicit path, that may well be different between tools, it creates more potential for divergence/tool-specific setup.

Can we also de-dupe the formatting info, remove it from the env var one, and replace with a reference to this, otherwise I recon we'll end up with it being out of sync over time.

Reply:
De-deuped and `@ref`'d in 36b139a2